wallet-api/src/Wallet/API.hs

```diff
@@ -401,18 +370,68 @@ ownPubKeyTxOut v = pubKeyTxOut v <$> fmap pubKey myKeyPair
 --   TODO: Also compute the fee
 createTxAndSubmit ::
     (Monad m, WalletAPI m)
-    => Set.Set TxIn
+    => SlotRange
+    -> Set.Set TxIn
     -> [TxOut]
     -> m Tx
-createTxAndSubmit ins outs = do
-    (vfrom, vto) <- validationInterval
+createTxAndSubmit range ins outs = do
     let tx = Tx
             { txInputs = ins
             , txOutputs = outs
             , txForge = 0
             , txFee = 0
-            , txValidFrom = vfrom
-            , txValidTo = vto
+            , txValidRange = range
             }
     submitTxn tx
     pure tx
+
+-- | An open-ended 'SlotRange' that begins at slot 1.
+defaultSlotRange :: SlotRange
+defaultSlotRange = $$(Interval.always)
```

Comment:
Hm, this is a bit annoying, but I'm not sure what else to do.

Reply:
Maybe this should be next to the `Interval` module and it should have `Interval` and `Interval.TH` or something?

Reply:
That would cause two additional imports in the tutorial and sample code (one for `Interval.TH` to be used in on-chain code, and one for `Interval` to be used in off-chain code). With the current solution we only need one additional import. Although the way you described would be cleaner than including it in `Wallet.API`.

Reply:
I'm happy for you to make the call here